### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DenVia**

### CODE REVIEW

File: app/test_main.py
Comment (LiudmylaKulinchenko) [inline app/test_main.py:1]: Add tests for negative numbers and incorrect types

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/test_main.py [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+    (28, 29, [3, 3]),
+    (100, 100, [21, 17])
+])
+def test_get_human_age(
+        cat_age: int,
+        dog_age: int,
+        expected_result: list
+) -> None:
+    if not isinstance(cat_age, int) or not isinstance(dog_age, int):
+        raise TypeError("Arguments should be int type")
```
Comment (LiudmylaKulinchenko) [inline app/test_main.py:23]: You should create a new test for incorrect types fot the tested function, not make actions in your test

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]
